getting-started/vs-code-integration/convert-project-wizard.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Convert to Telerik Project
+page_title: Convert to Telerik Project - Visual Studio Code Integration
+description: Learn how to convert your existing Blazor app to use Telerik components in it with a few clicks through our Visual Studio Code Templates.
+slug: getting-started-vs-code-integration-convert-project
+position: 3
+---
+
+# Convert to Telerik Application
+
+This article demonstrates how to enable the Progress&reg; Telerik&reg; UI for Blazor components in your existing Blazor app with a few mouse clicks.
+
+To add the Telerik Components to your existing Blazor application, use the **Convert Command** of the Telerik UI for Blazor Visual Studio Code Extension.
```

Comment:
The bold text should be `inline code` because this is a command.

Reply:
Done.